share/module_check_a_mundo.F

```diff
@@ -1351,6 +1351,63 @@ END FUNCTION bep_bem_ngr_u
          END IF
       END IF
 
+!-----------------------------------------------------------------------
+! If qna_update = 0, set io_form_auxinput17 to 0 so WRF will not try to
+! input the data; auxinput_interval must also be 0
+!-----------------------------------------------------------------------
+
+      IF ( model_config_rec%qna_update .EQ. 0 ) THEN
+         model_config_rec%io_form_auxinput17 = 0
+         DO i = 1, model_config_rec % max_dom
+            wrf_err_message = '--- NOTE: qna_update is 0, ' // &
+                  'setting io_form_auxinput17 = 0 and auxinput17_interval = 0 for all domains'
+            CALL wrf_debug ( 1, TRIM( wrf_err_message ) )
+            model_config_rec%auxinput17_interval(i)   = 0
+            model_config_rec%auxinput17_interval_y(i) = 0
+            model_config_rec%auxinput17_interval_d(i) = 0
+            model_config_rec%auxinput17_interval_h(i) = 0
+            model_config_rec%auxinput17_interval_m(i) = 0
+            model_config_rec%auxinput17_interval_s(i) = 0
+         ENDDO
+      ELSE
+         IF ( model_config_rec%io_form_auxinput17 .EQ. 0 ) THEN
+            wrf_err_message = '--- ERROR: If qna_update /= 0, io_form_auxinput17 must be /= 0'
+            CALL wrf_message ( wrf_err_message )
+            wrf_err_message = '--- Set io_form_auxinput17 in the time_control namelist (probably to 2).'
+         CALL wrf_debug ( 0, TRIM( wrf_err_message ) )
+         count_fatal_error = count_fatal_error + 1
+         END IF
```

Comment:
@twjuliano 
Tim,
Go ahead and mention that the interval needs to be non-zero, too.